Atomic/diff.py

```diff
@@ -455,7 +530,7 @@ def _print_diff(file_list):
             util.write_out("\nFiles only in {}:".format(right_docker_obj))
             _print_diff(self.right)
         if len(self.common_diff):
-            util.write_out("\nCommon files that are different:")
+            util.write_out("\nCommon files that are different: (reason)")
```

Comment:
That LOC is just to print the header(title).  The rest comes in the _print_diff method.